randommenu_platinum.rb

```diff
@@ -0,0 +1,66 @@
+# Random Menu Generator
+
+# create empty arrays
+adjective = []
+cook_method = []
+food_item = []
+
+# Get user input for how many item menus
+puts "Hey! Let's generate a random menu for you.\nHow many items would you like on your menu?"
+list_num = gets.chomp
+
+# Error message for invalid number
+until list_num.to_i.to_s == list_num && list_num.to_i > 0
+  puts "That's NOT a valid number! Choose again."
+  list_num = gets.chomp
+end
+
+# Convert user input from string to integer
+list_num = list_num.to_i
```

Comment:
You could do this on the same line as your `gets.chomp` and then just check that it's greater than zero on line 13.